applications/FluidDynamicsApplication/tests/EmbeddedPiston2DTest/EmbeddedPiston2DTestParameters.json

```diff
@@ -0,0 +1,187 @@
+{
+    "problem_data"     : {
+        "problem_name"  : "embedded_piston_2d_test",
+        "parallel_type" : "OpenMP",
+        "echo_level"    : 0,
+        "start_time"    : 0.0,
+        "end_time"      : 0.25
+    },
+    "output_processes" : {
```

Comment:
can you please avoid printing by default? This is slow and not needed in 99.9% of the cases this test is run.
Either comment it (e.g. by calling it `_output_processes`) or make a flag in the test-script to only add this then. => I did this in the ALEFluid-solver test in the meshmoving app

Reply:
I've put a flag in the test class that checks if the user wants to print or not. The unique overhead is that the process is always instantiated. 

In any case your approach is better since you don't even instantiate the process. Thanks!

Reply:
I've checked it, and in my case it would imply to rewrite the creation of processes in the auxiliary class I've created so I'd rather keep it as it is.